tests/output/ConsoleInteractionTest.py

```diff
@@ -439,8 +473,10 @@ def test_print_result_no_input(self):
                                  """
 Project wide:
 |    | [NORMAL] origin:
-|    | message
-""")
+|    | {}
+""".rstrip("\r\n").format(highlight("message", TextLexer(),
```

Comment:
The output doesn't match, because highlight seems to be adding a "\n" and then return a `str` object. So I thought the best way is to use `rstrip()` to get rid of it.